src/compiler/types.ts

```diff
@@ -505,6 +505,7 @@ namespace ts {
         JSDoc                                         = 1 << 21, // If node was parsed inside jsdoc
         /* @internal */ Ambient                       = 1 << 22, // If node was inside an ambient context -- a declaration file, or inside something with the `declare` modifier.
         /* @internal */ InWithStatement               = 1 << 23, // If any ancestor of node was the `statement` of a WithStatement (not the `expression`)
+        JsonFile                                      = 1 << 24, // If node was parsed in a Json

```

Comment:
i would just call it `JSON`

Reply:
or `Json` for consistency

Reply:
https://github.com/Microsoft/TypeScript/pull/22167/files#diff-4b8bd1eea29904f1be39cd864e1a45c0R489 has this as JavaScriptFile isnt it more consitent to use JsonFile ?